### ISSUE TITLE
Add feature file uri to step results

### ISSUE THREAD

**Issue by timhemming-fico** [eugene-sea/karma-cucumber-js#11]
Would it be possible to add the feature file uri to the StepResult in [CucumberKarmaListener#hear](https://github.com/eugene-sea/karma-cucumber-js/blob/master/src/cucumber-karma-listener.ts#L33)?

For example:
```
const stepId = `${description} : file ${this.feature.getUri()} : line ${step.getLine()}`;
```

This would help navigating to the corresponding feature files from our CI server result pages.

**Comment by eugene-sea** [comment]
OK, I will add feature file URI

**Comment by eugene-sea** [comment]
I have published karma-cucumber-js@1.0.1

**Comment by timhemming-fico** [comment]
Thanks for the quick response. This works great for us.
